sympy/matrices/solvers.py

```diff
@@ -441,10 +441,10 @@ def _gauss_jordan_solve(M, B, freevar=False):
         The right hand side of the equation to be solved for.  Must have
         the same number of rows as matrix A.
 
-    freevar : List
+    freevar : boolean
```

Comment:
Can you change the description for this, because this one works better for `free_var_index` IMO?
Looks good otherwise

Reply:
Updated the description.